src/lib.rs

```diff
@@ -25,22 +31,30 @@
 //!
 //! # Example
 //!
-//! This example will show how to print "Hello, world!" on the host.
+//! ## Using `hio::HStdout`
 //!
-//! Target program:
+//! This example will demonstrate how to print formatted strings.
 //!
-//! ```
+//! ```rust
 //! #[macro_use]
```

Comment:
This is not required anymore. Could you remove it from this and the other example?